README.md

```diff
@@ -62,25 +62,29 @@ yarn link @code-dot-org/ml-playground
 
 This will set up a symlink in main repo's `apps/node_modules/` to point at your local changes. 
 
-Run 
 
-```
-yarn run build
-```
-
-in this repo, and then the main repo's `apps` build should pick the changes up next time it builds.
+### Building changes for a local Code Studio
+In main repo:
```

Comment:
I see the desire to add more main repo instructions, but I did want to have a section dedicated to the only build step usually required rebuild this app for consumption in that repo (which is `yarn run build`)...

Reply:
Sounds good! I added it back in.